organization/board-of-directors-meetings/2022-02-28-minutes.md

```diff
@@ -0,0 +1,29 @@
+---
+type: Resource
+---
+
+# Board meeting February 28th 2022
+
+1. Review action items:
+   1. Action: Eric to calculate the effect of the suggested salary policy on the burn rate
+      1. Unchanged: Blocked on needing a meeting with the bookkeeper
+   2. Action: Ben will revise the statement of support or partnership agreement based on the one sent to Montreal for approval by the board
+      1. Follow-up: Re-approach partner onboarding process.
+   3. Action: Eric and Boris to renegotiate with Multiple Ventures BV for services in 2022
+      1. Follow up: Eric and Boris to review the proposal sent by Multiple Ventures BV
+   4. Action: Ben with help from Eric and Boris will work on a good framing for our work in North America in Quarter 1
+      1. Completed
+   5. Action: Course request, Boris to review the request with the submitter based on planning
+      1. Completed: coursework approved in line with what we discussed
+   6. Action: Boris to engage law firm based on his email
+      1. Completed
+   7. In April: Discuss who to hire next
+   8. When the lockdown in the Netherlands ends: Boris to resume recruiting the ‘medewerker administratie’
+2. For discussion:
+   1. Incoming documentation requests
+      1. Decision: we can re-engage external council
```

Comment:
Is this referring to the strategic council or some other body? It looks very cryptic right now. If it needs to be secret, perhaps make that explicit like "re-engage [organization name redacted]"?

Reply:
Great point, what about:

```suggestion
      1. Decision: we can re-engage external legal council
```

Reply:
Yes, then it makes more sense!